docs/docs/self-hosting/manage/production.md

```diff
@@ -90,7 +90,7 @@ but in the Projections.Customizations.Telemetry section
 ### Prefer CockroachDB
 
 ZITADEL supports [CockroachDB](https://www.cockroachlabs.com/) and [PostgreSQL](https://www.postgresql.org/).
-We highly recommend using CockroachDB,
+We recommend using CockroachDB,
```

Comment:
We recommend to chose the database depending on the use case, right?

Reply:
At the moment we have a better understanding of CRDB than psql because we didn't have any trouble with psql thats why I haven't changed it.